example/src/CustomCardScreen.js

```diff
@@ -108,6 +140,12 @@ const styles = StyleSheet.create({
     color: '#333333',
     marginBottom: 5,
   },
+  optionalParams: {
+    fontSize: 10,
```

Comment:
Font size is too small, and different of mandatory fields, may be we should create common style for mandatory and optional fields?